package.json

```diff
@@ -91,6 +91,7 @@
     "style-dictionary": "3.7.1",
     "ts-jest": "^29.0.3",
     "ts-node": "^8.10.2",
+    "tsconfig-paths": "^4.1.0",
```

Comment:
Is this specifically needed for the node_module imports?

Reply:
No, for any import with aliases when running `ts-node`

Reply:
ah okay, I hadn't run into this in the past.. good to know. Bit annoying it doesn't just respect the paths settings in tsconfig 😞 .